app/src/main/java/com/guruthedev/instagram/MainActivity.kt

```diff
@@ -1,20 +1,29 @@
 package com.guruthedev.instagram
 
+import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import androidx.databinding.DataBindingUtil
 import androidx.navigation.findNavController
 import com.guruthedev.instagram.databinding.ActivityMainBinding
+import com.guruthedev.instagram.ui.fragments.auth.login.LoginFragment
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
-
+    private lateinit var sharedPreference: SharedPreference
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = DataBindingUtil.setContentView(this, R.layout.activity_main)
-    }
+        sharedPreference =  SharedPreference(this)
 
+        if (sharedPreference.isLoggedIn()){
```

Comment:
This is not a correct way to Handle loggedIn scenario.
What does the below code actually do?

Reply:
sorry this code not working